pyanaconda/ui/tui/spokes/installation_source.py

```diff
@@ -271,7 +271,7 @@ def refresh(self, args=None):
         self._container = ListColumnContainer(1)
 
         dialog = Dialog(_("Repo URL"))
-        self._container.add(EntryWidget(dialog.title, self.data.method.url), self._set_repo_url, dialog)
+        self._container.add(EntryWidget(dialog.title, self._url), self._set_repo_url, dialog)
```

Comment:
Would it be the same?

The `self._url` is set in init by:

```
self._url = self.data.url.url
```
which I guess could be different in some situations.

Reply:
I am not sure what you mean. `SpecifyRepoSpoke` is created every time we want to set url, so `self._url` always holds the current value. In case that the current installation method is different, it still makes sense to use `self.data.url.url`, because it is last known value of an attribute we are trying to set.

Reply:
Ok, then. It looks like it should work :+1: